bin/acpp

```diff
@@ -925,10 +935,51 @@ class acpp_config:
   def is_pure_linking_stage(self):
     return len(self.source_file_arguments) == 0
 
-def run_or_print(command, print_only):
+def filter_cmd_args(command, verbose = False):
+  new_cmd = []
+
+  whitelist = [
+    "-I", "-D", "-W", "-std=","-pedantic-errors"
+  ]
+
+  whitelist_enable_next = [
+    "-isystem", "-o", "-c"
```

Comment:
`-I` can also be used in conjunction with the next argument, e.g. `-I /path/to/includes`

Reply:
done ! I documented it also in the implementation